react-native/src/RealmContext.js

```diff
@@ -0,0 +1,6 @@
+import {createRealmContext} from '@realm/react';
+import { Task } from './TaskSchema';
+
+export default createRealmContext({
+  schema: [Task.schema],
+});
```

Comment:
Is the idea behind separating the createRealmContext & schema so that they can be presented as 2 different steps in the tutorial?

If not, it might be worth keeping them combined as one file since it functionally does the same thing, and is a bit simpler. I think it gives the template-app-user one single blackbox (until they want to learn more) rather than two blackboxes. But that's just a suggestion and if you prefer it this way, feel free to keep it this way.

Reply:
Yeah it will be a new step in the tut but I think this makes sense in the template as well. Co-locating with `TaskSchema` is simpler for just one schema but once you add a second schema then you'd want to refactor to something like this. Given that a lot of apps will use multiple schemas I think we should just give them the pattern up front.